pkg/kubelet/cm/dra/plugin/noderesources.go

```diff
@@ -252,16 +254,29 @@ func (c *nodeResourcesController) run(ctx context.Context) {
 	// For now syncing starts immediately, with no DeleteCollection. This
 	// can be reconsidered later.
 
-	// While kubelet starts up, there are errors:
-	//      E0226 13:41:19.880621  126334 reflector.go:150] k8s.io/client-go@v0.0.0/tools/cache/reflector.go:232: Failed to watch *v1alpha2.ResourceSlice: failed to list *v1alpha2.ResourceSlice: resourceslices.resource.k8s.io is forbidden: User "system:anonymous" cannot list resource "resourceslices" in API group "resource.k8s.io" at the cluster scope
-	//
-	// The credentials used by kubeClient seem to get swapped out later,
-	// because eventually these list calls succeed.
-	// TODO (https://github.com/kubernetes/kubernetes/issues/123691): can we avoid these error log entries? Perhaps wait here?
+	// Wait until we're able to get a Node object.
+	// This means that the object is created on the API server,
+	// the kubeclient is functional and the node informer cache is populated with the node object.
+	// Without this it doesn't make sense to proceed further as we need a node name and
+	// a node UID for this controller to work.
+	var node *v1.Node
+	var err error
+	for {
+		node, err = c.getNode()
+		if err == nil {
+			break
+		}
+		logger.V(5).Error(err, "getting Node object failed, waiting")
```

Comment:
Just noticed one more thing. There is no `V(5).Error`, is is the same as `Error`.

```suggestion
		logger.V(5).Info("getting Node object failed, waiting", "err", err)
```